client/ayon_core/addon/base.py

```diff
@@ -67,6 +68,56 @@
 }
 
 
+class ProcessPreparationError(Exception):
+    """Exception that can be used when process preparation failed.
+
+    The message is showed to user (either as UI dialog or printed). If
+        different error is raised a "generic" error message is showed to user
+        with option to copy error message to clipboard.
```

Comment:
```suggestion
    The message is shown to user (either as UI dialog or printed). If
        different error is raised a "generic" error message is shown to user
        with option to copy error message to clipboard.
```

Reply:
I'll wait until you go through it so I can do one big batch commit 🙂 .

Reply:
Done.